modules/graceful/net_unix.go

```diff
@@ -150,11 +150,13 @@ func CloseProvidedListeners() error {
 	return returnableError
 }
 
+var GetListener = DefaultGetListener
```

Comment:
It needs some comments about why it is done so here.

Otherwise it looks good to me.

Reply:
Primary reason for it being there is because that is where the GetListener function was before and I was trying to avoid moving anything or otherwise avoid making changes that would be cosmetic or unnecessary. Putting it there kept the whole set of changes to just the 2 lines and makes it very clear that all I did was make a function replaceable. I could move the GetListener variable declaration to the top of the file and it would have the same effect obviously, and if that's preferred I will.

Reply:
I moved the GetListener to the top of the file with the other vars, commented both GetListener and DefaultGetListener to explain their usage, and added a GetListener/DefaultGetListener pair to net_windows.go